docs/settings.md

```diff
@@ -111,3 +111,4 @@ Legend:
 | `conversations_files`                | string<br>`1` or `0`                                             | `1`        | No   | 🖌️    | Whether the files app integration is enabled allowing to start conversations in the right sidebar                                                                                                                                                                                |
 | `conversations_files_public_shares`  | string<br>`1` or `0`                                             | `1`        | No   | 🖌️    | Whether the public share integration is enabled allowing to start conversations in the right sidebar on the public share page (Requires `conversations_files` also to be enabled)                                                                                                |
 | `enable_matterbridge`                | string<br>`1` or `0`                                             | `0`        | No   | 🖌️    | Whether the Matterbridge integration is enabled and can be configured                                                                                                                                                                                                            |
+| `inactivity_lock_time`               | int                                                              | `0`        | No   |        | A duration (in seconds) after which rooms are locked. Calculated from the last activity in the room.                                                                                                                                                                             |
```

Comment:
since the background jobs are only run hourly (I would even change it to daily),
I would make this config "days" of inactivity

Reply:
Makes sense